apps/builder/app/builder/shared/expression-editor.tsx

```diff
@@ -341,7 +341,9 @@ const emptyScope: Scope = {};
 const emptyAliases: Aliases = new Map();
 
 const wrapperStyle = css({
-  "& .cm-content": {
+  // do not override .cm-content here
+  // it may break scrolling
```

Comment:
please describe what the problem is we discovered, we won't be able to recall it later

Reply:
ideally ofc we have something reusable that has only a single place  where these classes are used at all ... these 2 places will be 5 places later

Reply:
I would love for it to be a component that  we can control over props and all those classes are hidden inside of it, working with those global classes shouldn't be necessary in react

Reply:
e.g. you could expose the necessary option via css variable if prop doesn't work